DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/lacp/test_lacp_no_traffic.py

```diff
@@ -0,0 +1,125 @@
+        err_msg = f'Bond bond_{unsupported_modes.index(mode)} with unsupported mode {mode} is in `UP` state'
+        assert out[0][device]['parsed_output'][0]['operstate'] == 'DOWN', err_msg
+
+
+async def test_lacp_max_lags(testbed):
+    """
+    Test Name: LACP maximum lags
+    Test Suite: suite_functional_lacp
+    Test Overview: Test lacp maximum lags creation
+    Test Procedure:
+    1. Create max 'number of  bond devices
+    2. Verify it is possible to create max 'number of bond devices
+    2. Enslave all ports to different bonds
+    3. Verify it is possible to enslave all ports to bond(s)
+    """
+    dent_devices = await tb_get_all_devices(testbed)
+    device = dent_devices[0].host_name
+    max_bonds = 64
+    # 1. Create max 'number of  bond devices
+    cmd = [{'device': f'bond_{idx}',
+            'type': 'bond',
+            'mode': '802.3ad'} for idx in range(max_bonds)]
+    out = await IpLink.add(input_data=[{device: cmd}])
+    # 2. Verify it is possible to create bond devices
+    assert out[0][device]['rc'] == 0, 'Fail to create bond(s)'
+
+    _, num_of_ports = await dent_devices[0].run_cmd('ifconfig  -a | grep -Eo "swp[0-9]+" | wc -l')
+    assert int(num_of_ports) > 1, 'Fail getting port on devices'
+
+    out = await IpLink.set(input_data=[{device: [
+        {'device': f'swp{idx+1}',
+         'operstate': 'down'} for idx in range(int(num_of_ports))]}])
+    assert out[0][device]['rc'] == 0, 'Fail to set port(s) to down state'
+
+    # 2. Enslave all ports to different bonds
+    out = await IpLink.set(input_data=[{device: [
+        {'device': f'swp{idx+1}',
+         'master': f'bond_{idx}'} for idx in range(int(num_of_ports))]}])
+
+    # 3. Verify it is possible to enslave all ports to bond(s)
+    assert out[0][device]['rc'] == 0, 'Fail to enslave port(s) to bond'
+
+    out = await IpLink.set(input_data=[{device: [
+        {'device': f'swp{idx+1}',
+         'operstate': 'up'} for idx in range(int(num_of_ports))]}])
+    assert out[0][device]['rc'] == 0, 'Fail to set bond(s) to up state'
+
+
+async def test_lacp_max_ports_in_lags(testbed):
+    """
+    Test Name: LACP ports in lag
+    Test Suite: suite_functional_lacp
+    Test Overview: Test can enslave max number of ports to lag
+    Test Procedure:
+    1. Create bond device
+    5. Add 9 ports to bond
+    6. Verify only 8 ports can be enslaved to bond
+    """
+    dent_devices = await tb_get_all_devices(testbed)
+    device = dent_devices[0].host_name
+    bond = 'bond_1'
+    max_ports_in_lag = 8
+
+    # 1. Create bond device
+    out = await IpLink.add(input_data=[{device: [{'device': bond,
+                                                  'type': 'bond',
+                                                  'mode': '802.3ad'}]}])
+    assert out[0][device]['rc'] == 0, 'Fail to create bond'
+
+    # 5. Add 9 ports to bond
+    out = await IpLink.set(input_data=[{device: [
+        {'device': f'swp{idx+1}',
```

Comment:
I would suggest getting the port list via `IpLink.show()`, and not using an `'swp'` string. The port on DUT might have a different naming convention (`sw1p1`).